<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LH7U05CG6375FW6" name="com.optumrx.irisuiuxqe.test.OrderCreationTest" tests="2" failures="0" timestamp="12 Mar 2018 12:07:37 GMT" time="71.904" errors="1">
  <testcase name="oc_RefilRx_01" time="44.855" classname="com.optumrx.irisuiuxqe.test.OrderCreationTest"/>
  <testcase name="oc_RefilRx_02" time="27.049" classname="com.optumrx.irisuiuxqe.test.OrderCreationTest">
    <error type="java.lang.IllegalArgumentException" message="Cannot find elements when the XPath expression is null.">
      <![CDATA[java.lang.IllegalArgumentException: Cannot find elements when the XPath expression is null.
	at org.openqa.selenium.By.xpath(By.java:113)
	at com.optumrx.irisuiuxqe.util.ComponentMethods.clickOnWebElelent(ComponentMethods.java:56)
	at com.optumrx.irisuiuxqe.ordercreation.HomePage.clickRxHistory(HomePage.java:31)
	at com.optumrx.irisuiuxqe.test.OrderCreationTest.oc_RefilRx_02(OrderCreationTest.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </error>
  </testcase> <!-- oc_RefilRx_02 -->
</testsuite> <!-- com.optumrx.irisuiuxqe.test.OrderCreationTest -->
